lib/features/booking/widgets/start_time_selection_widget.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:hatspace/data/inspection.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/features/booking/view_model/cubit/add_inspection_booking_cubit.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/route/router.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+import 'package:hatspace/theme/widgets/hs_text_field.dart';
+import 'package:hatspace/theme/widgets/hs_time_picker.dart';
+
+class StartTimeSelectionWidget extends StatelessWidget {
+  final ValueNotifier<StartTime?> startTimeNotifer;
+  final FocusNode startTimeFocusNode;
+  final List<int> minutesList;
+  final List<int> hourList;
+  const StartTimeSelectionWidget(
+      {required this.startTimeNotifer,
+      required this.startTimeFocusNode,
+      required this.minutesList,
+      required this.hourList,
+      super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final ValueNotifier showErrorMessages = ValueNotifier(false);
```

Comment:
do not create ValueNotifier in build method

Reply:
I removed it. Already implement bloc to handle it